keps/sig-storage/2923-csi-migration-ceph-rbd/README.md

```diff
@@ -35,6 +41,32 @@ used as described in its parent KEP. For all other contents, please refer to the
   on kubelet and treat Ceph RBD CSI migration as already complete. And directly redirect traffic to CSI 
   driver for all rbd related operations.
 
+## Design Details
+
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+No additional tests are needed, rather the issue is orchestrating CSI driver
+deployment for prow jobs. This means that it is not possible to run any test for
+RBD in k/k repository.
+
+##### Unit tests
+
+N/A
+
+##### Integration tests
+
+N/A
+
+##### e2e tests
+
+Support for tests after RBD migration will be performed by the subjected
+rbd csi driver.
```

Comment:
Please add links to added e2e tests, like shown in this PR: https://github.com/kubernetes/enhancements/pull/3351/files.

See requirement here: https://github.com/kubernetes/enhancements/commit/a86942e8ba802d0035ec7d4a9c992f03bca7dce9#diff-3df430c68270e73ed3fdb8880e18ca3b3a519c544bee244361db12ad014e0775R322

Reply:
You can probably point to existing CSI migration e2e tests in-tree and in RBD CSI Driver repo.

Reply:
Sure.. let me add those as well.. thanks.